runtime/bcverify/staticverify.c

```diff
@@ -1301,6 +1301,10 @@ checkStackMap (J9CfrClassFile* classfile, J9CfrMethod * method, J9CfrAttributeCo
 						delayedErrorCode = FALLBACK_VERIFY_ERROR;
 						exceptionDetails->stackmapFrameIndex = (I_32)j;
 						exceptionDetails->stackmapFrameBCI = (U_32)offset;
+#if JAVA_SPEC_VERSION >= 18
```

Comment:
Is there a test that fails before JDK18?

Reply:
Nope given the check for CFR is still required before JDK18 (verified in personal builds). So we only enable it in JDK18 and upwards.